### DISCUSSION TITLE
DNS-over-HTTPS

### DISCUSSION THREAD

**Discussion by ghost** [Alex313031/Thorium-Win#286] [Q&A]
Hi, is it possible to use DNS-over-HTTPS in Thorium-Win? 
I installed the .exe file, but for the DoH option it says that the Browser is managed, and it is greyed out.

I want to use Cloudflare DNS directly in the Browser because it maybe boosts the speed a bit as Cloudflare has the lowest response time of all.

**Comment by gz83** [comment]
Please try turning off secure DNS for your browser and turning off Thorium DNS in chrome://flags.

**Comment by ghost** [comment]
I think this occurs is because my PC is managed by a Domain Controller (AD). I have the same with GChrome.
